src/main/java/com/exadel/discount/model/security/Role.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.discount.model.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+@Getter
+public enum Role {
+    USER(Set.of(Permission.VENDORS_ORDINARY_ACCESS)),
+    ADMIN(Set.of(Permission.VENDORS_ORDINARY_ACCESS, Permission.VENDORS_PRIVILEGED_ACCESS));
+
```

Comment:
ups...  We have exactly  the same class in Entity folder..  Is it ok?

Reply:
The answer was given in the Skype